Sources/HMSRoomKit/Prebuilt/Internal/HMSPrebuiltConferenceView.swift

```diff
@@ -254,6 +258,16 @@ struct HMSPrebuiltConferenceView: View {
                 HMSPreviewRoleScreen()
             }
         }
+        .onAppear() {
+            if !roomModel.isNoiseCancellationEnabled && roomInfoModel.isNoiseCancellationOnByDefault {
```

Comment:
would something like this be a problem:
user turns of NC a refresh of the view happens, NC gets reenabled?

Reply:
The view refresh doesn't cause onAppear callback. it would be called when conference screen is removed and added back